Source/FikaAmazonAPI/AmazonSpApiSDK/Runtime/Utils.cs

```diff
@@ -30,6 +31,16 @@ public static string UrlEncode(string data)
             return encoded.ToString();
         }
 
+        /// <summary>
+        ///  Decode
+        /// </summary>
+        /// <param name="data"></param>
+        /// <returns></returns>
+        public static string UrlDecode(string data)
```

Comment:
do we need to use this ?

Reply:
This method is to solve the problem of URL secondary encoding.

> #35

Reply:
Do you test this ?

Reply:
Yes, I have verified it by testing with a lot of data

Reply:
Good